integration-tests/cucumber-tests-platform-common/src/test/resources/features/common/osgp-adapter-ws-core/DeviceInstallation/UpdateDevice.feature

```diff
@@ -23,11 +23,37 @@ Feature: CoreDeviceInstallation Device Updating
       | TechnicalInstallationDate | 2016-12-07T09:10:33.684 |
       | UsePrefix                 | false                   |
       | Metered                   | false                   |
-    Then the update device response is successfull
-    # Note: Alias is empty after update
+    Then the update device response is successful
     And the device exists
       | DeviceIdentification | TEST1024000000001 |
-  #		| Alias                | AfterTest         |
+      | Alias                | AfterTest         |
+
+  Scenario: Updating device data does not change GPS coordinates ( FLEX-4503 )
+    Given a device
+      | DeviceIdentification | TEST1024000000001 |
+      | Alias                | Alias             |
+      # Default values for the GPS coordinates are null.
+    When receiving an update device request
+      | DeviceIdentification      | TEST1024000000001       |
```

Comment:
Can we remove some of the fields? If they're not essential or interesting for this feature, we can use the defaults.